### PR TITLE
gerbv: new package

### PR DESCRIPTION
**Pull request by mogorman**
pcb gerber viewer.

### CODE REVIEW

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#     sleep 5
+#     echo hey mog
+#     cd po
+#     msginit -l ru
+#     cd ..
+#     touch man/gerbv.ru.1.in.po
+#   '';
+
+  meta = with stdenv.lib; {
+    description = "Gerbv -- A Gerber (RS-274X) viewer";
```
Comment (pSub) [inline pkgs/applications/science/electronics/gerbv/default.nix]: `description` should not contain the package name.

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    homepage = http://gerbv.geda-project.org/;
+    maintainers = with maintainers; [ mog ];
+    platforms = platforms.linux;
+    license = licenses.gpl2;
+    priority = 10;
```
Comment (pSub) [inline pkgs/applications/science/electronics/gerbv/default.nix]: Why do you need priority? Would be nice to document this, IMO.

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    sha256 = "1l2x8sb1c3gq00i71fdndkqwa7148mrranayafqw9pq63869l92w";
+  };
+
+  buildInputs = [ pkgconfig gettext libtool automake autoconf cairo gtk desktop_file_utils ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+#    postConfigure = ''
```
Comment (pSub) [inline pkgs/applications/science/electronics/gerbv/default.nix]: Why is that commented?

File: pkgs/applications/science/electronics/gerbv/default.nix [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  src = fetchgit {
+    url = git://git.geda-project.org/gerbv.git;
+    rev = "76b8b67bfa10823ce98f1c4c3b49a2afcadf7659";
+    sha256 = "1l2x8sb1c3gq00i71fdndkqwa7148mrranayafqw9pq63869l92w";
+  };
+
+  buildInputs = [ pkgconfig gettext libtool automake autoconf cairo gtk desktop_file_utils ];
+
+  preConfigure = ''
+    ./autogen.sh
```
Comment (pSub) [inline pkgs/applications/science/electronics/gerbv/default.nix]: There is `autoreconfHook` to automate this.

File: pkgs/applications/science/electronics/gerbv/default.nix (hunk 2 above)
Comment (mogorman) [inline pkgs/applications/science/electronics/gerbv/default.nix]: when i built without that i got an error. in googling I saw someone setting that priority.  so i copy pasted.  don't know if its right thing to do

**Comment by jagajaga** [comment]
Please properly name commit following our guide in `CONTRIBUTING.md`.

**Comment by mogorman** [comment]
not sure if i edited correctly but i think this matches what you want

File: pkgs/applications/science/electronics/gerbv/default.nix (hunk 2 above)
Comment (jagajaga) [inline pkgs/applications/science/electronics/gerbv/default.nix]: What kind of error do you get?

Comment (mogorman) [inline pkgs/applications/science/electronics/gerbv/default.nix]: nix-env -f $NIXPKGS  -i gerbv
replacing old ‘gerbv-2015-10-08’
installing ‘gerbv-2015-10-08’
building path(s) ‘/nix/store/1cyz0acybk9440ylw69s72xiq75wyh12-user-environment’
Wide character in die at /nix/store/jjgyvh88yw37gg5ad5lj1z0qdv31d7jy-buildenv.pl line 79.
collision between ‘/nix/store/q8d3c4fhjr1sxhbljdcxfm7s19zjlvzj-hexchat-2.10.2/share/applications/mimeinfo.cache’ and ‘/nix/store/5p57j611z8nj01axvk68qbcqjj0xaja0-gerbv-2015-10-08/share/applications/mimeinfo.cache’; use ‘nix-env --set-flag priority NUMBER PKGNAME’ to change the priority of one of the conflicting packages
builder for ‘/nix/store/7izpjwdqapidp8ik9ff231bs31xrmk37-user-environment.drv’ failed with exit code 2
error: build of ‘/nix/store/7izpjwdqapidp8ik9ff231bs31xrmk37-user-environment.drv’ failed

**Comment by matthiasbeyer** [comment]
@mogorman I guess `init at <commit hash>` is more appropriate, am I right @jagajaga ?

**Comment by jagajaga** [comment]
@matthiasbeyer no, date aka version is what we need :)

**Comment by mogorman** [comment]
can someone tell me what i need to change to get this in?

File: pkgs/applications/science/electronics/gerbv/default.nix (hunk 2 above)
Comment (jagajaga) [inline pkgs/applications/science/electronics/gerbv/default.nix]: We need to remove that file during patch phase.

**Comment by mogorman** [comment]
fixed like suggested in other package i made

**Pull request merged**